packages/IMAPClient-UI.package/ICFolderDialog.class/instance/updateProcess.st

```diff
@@ -2,11 +2,10 @@ action - emails
 updateProcess
 		
 	| successful |
-	successful := true.
-	[self emailAccounts do: [:anICEndpoint | successful := ( successful and: (
-    anICEndpoint rootFolder updateAndShowProgressWith: [self progressCallback]))]]
-  on: ConnectionClosed
+	successful := false.
+	[self emailAccounts do: [:anICEndpoint | successful := ( successful and: (anICEndpoint rootFolder updateAndShowProgressWith: [self progressCallback]))]]
+  on: NetworkError
 	do: [:e | self inform: e asString].
   
-	successful ifTrue: [self updateSynchTime].
+	successful ifFalse: [self updateSynchTime].
```

Comment:
Why was this changed? Also, that variable is now named misleadingly.

Reply:
You're right. That's an obvious mistake.

Reply:
Fixed/reverted :)